ui/position.js

```diff
@@ -142,12 +142,13 @@ $.position = {
 	getWithinInfo: function( element ) {
 		var withinElement = $( element || window ),
 			isWindow = $.isWindow( withinElement[ 0 ] ),
-			isDocument = !!withinElement[ 0 ] && withinElement[ 0 ].nodeType === 9;
+			isDocument = !!withinElement[ 0 ] && withinElement[ 0 ].nodeType === 9,
+			hasOffset = element && ( !isWindow && !isDocument );
```

Comment:
Because the fact that we were using `withinElement` IS THE BUG it the entire reason for the change. `withinElement` can be window or document and both of these must be guarded against as they will throw errors. Hence the check your commenting on.